proto/meta.proto

```diff
@@ -196,6 +196,17 @@ message ActivateWorkerNodeResponse {
   common.Status status = 1;
 }
 
+message ListAllNodesRequest {
```

Comment:
I think the `GetClusterInfoResponse` structure in our code already includes this semantics, doesn't it?

Reply:
Maybe. I would still leave it. I did not change the `ListAllNodes` messages. They were added by @wyhyhyhyh [in this PR](https://github.com/risingwavelabs/risingwave/commit/b3225e5dce323fa90c31e6bf4bed1a444a6269da). GitHubs diff is just a bit confusing here. I will try to clarify the diff a bit by moving the messages around. Let's see if it works... 

My suggestion is that we keep the messages for now as is. IMHO changing them should be its own PR, since it is not related to cordon.